dashboard/test/ui/runner.rb

```diff
@@ -430,12 +430,10 @@ def how_many_reruns?(test_run_string)
 $errbrowserfile.close
 
 $suite_duration = Time.now - $suite_start_time
-$average_test_duration = $suite_duration / ($suite_success_count + $suite_fail_count)
 
 HipChat.log "#{$suite_success_count} succeeded.  #{$suite_fail_count} failed. " +
   "Test count: #{($suite_success_count + $suite_fail_count)}. " +
   "Total duration: #{format_duration($suite_duration)}. " +
```

Comment:
you need to remove the trailing + from this line

it's actually valid syntax but it will complain because the next statement will return nil